server/admin_management/tests/test_data/access_control_privilege_data.py

```diff
@@ -0,0 +1,147 @@
+
+from api.app.schemas.pagination import PagedResultsSchema
+from api.app.schemas.schemas import FamAccessControlPrivilegeGetResponse
+
+APP_DELEGATED_ADMIN_PAGED_RESULT_4_RECORDS = PagedResultsSchema[FamAccessControlPrivilegeGetResponse](
+    **{
```

Comment:
learned something cool, I didn't know this existed

Reply:
:) This is like Javascript destructuring with `...rest`. It was used in several places in FAM but I kept forgetting to use it and also am confused since python has single **"`*`"** and double **"`**`"** on different types of objects.